tests/unit_test/data_processor/upload_handler_log_processor_test.py

```
@@ -0,0 +1,144 @@
+        result = UploadHandlerLogProcessor.is_event_in_progress("test_bot", "test_collection_2")
+        assert not result
+
+    def test_add_log_success(self):
+        bot = 'test_bot'
+        user = 'test_user'
+        UploadHandlerLogProcessor.add_log(bot, user, collection_name="test_collection", event_status=EVENT_STATUS.SAVE.value)
+        UploadHandlerLogProcessor.add_log(bot, user,
+                                         status='Success',
+                                         event_status=EVENT_STATUS.COMPLETED.value)
+        log, count = BaseLogHandler.get_logs(bot, "file_upload")
+        assert not log[0].get('exception')
```

Ruff (0.12.2)

Use of `assert` detected

(S101)